### DISCUSSION TITLE
💡 Silence non-essential notifications

### DISCUSSION THREAD

**Discussion by xeruf** [Manjaro-Sway/manjaro-sway#814] [Ideas]
### Summary

I almost all the time silence notifications because I get plenty that annoy me (charging: https://github.com/manjaro-sway/manjaro-sway/issues/480, messages, ...) but I would like to keep some basic notifications, most importantly: Battery being almost empty ^^

### Drawbacks

Not sure how those can be reliably discerned in an unbiased way

### Implementation idea

have the notification button cycle through multiple modes

### Will you be able to contribute a PR for this?

- [ ] Yes I will be able to get that feature a *lot* quicker.

**Comment by xeruf** [comment]
@boredland what handles notifications? is it possible to have a dismiss / dismiss all button for them?

**Reply by boredland** [reply]
There is a mute all/DND mode for messages already. Do you mean such a thing?

**Reply by xeruf** [reply]
no, two things:
- dismiss notifications individually (sometimes it is just in the way)
- more granular DND

**Reply by xeruf** [reply]
As for DND, urgency levels should be considered - battery low/critical is a high-urgency notification (which at least `notify-send` allows to set), but currently there is no distinction made for urgency levels, neither in filtering nor appearance

**Comment by xeruf** [comment]
@boredland how can I send a notification from the terminal in an adhoc script? `notify-send` does not seem to do anything.

**Reply by xeruf** [reply]
@boredland ping;)

**Reply by boredland** [reply]
For me notify-send just works?

**Reply by xeruf** [reply]
oh right it does now, interesting
